Documentation/Signoffs/OEM Component- Auto_Light-signoff.md

```diff
@@ -0,0 +1,99 @@
+
+## Analysis
+-----
+From the Datasheet
+
+![LVT 817](https://user-images.githubusercontent.com/101354063/206586290-37ae6b0b-b3ca-41a7-96c7-75b4382fc6b4.PNG)
+
+Forward Current = 40 mA  for opto-isolation [3]
+
+The Voltage divider will need to produce a maximum output 5 V and 40 mA.
+
+### Voltage divider
+
+Vout = Vin ${ R2 \over R1+R2}$
+
+5V = 12V ${ 125 \ ohms \over 250 \ ohms  + 125 \ ohms}$
+
+$Total \ Resistance: 375 \ ohms = {250 \ ohms +125 \ ohms}$ 
+
+$Max \ current: 40\ mA = {10V \over 375 \ ohms }$
+
+$Min \ current: 26.67\ mA = {10V \over 375 \ ohms }$
+
+Test different voltages going through Voltage Divider(0 to 15)
+
+![image](https://user-images.githubusercontent.com/101354063/206635438-b9d504b8-7e54-471c-8ddc-083a5be3e416.png)
+
+Test different voltages going through Voltage Divider(10 to 15)
+
+![image](https://user-images.githubusercontent.com/101354063/206637463-7d53790e-b779-426a-9310-c4f548f22864.png)
+ 
+## Master microcontroller communication
+
+A wire will be connected directly to the optoisolator to the digital pin of the master microcontroller. The microcontroller receives voltage from themselves into digital pins, and the optoisolator controls whether the voltage passes through or not based on whether the auto light is on or off. Also, the master microcontroller has availible GPIO pins to shorten the delay of input into the communication network with less items to pass through. With the voltage being either 5 V or 0, the microcontroller will use the digital GPIO pins to read the voltage.
+
+
+### Summary
+The voltage going through the headlight is 12 volts and the current is in the range of 4.0 to 4.7 amp based on research using the auto light diagram from Figure 1 and the Sylvania H13 XtraVision Halogen Headlight Bulb [1]. Furthermore, based on the information, I gather that the current won't affect the device, but the voltage will.
```

Comment:
What does this mean? How will current not affect the "device"? What is the device to which you refer? 

I'm just not clear on what is meant here.